Description: Removed crashing blockdiag doc line
 This line is making blockdiag stack dump. Until a better fix can be found,
 we just remove that.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2018-02-20

Index: nova/doc/source/reference/notifications.rst
===================================================================
--- nova.orig/doc/source/reference/notifications.rst
+++ nova/doc/source/reference/notifications.rst
@@ -342,7 +342,6 @@ Existing versioned notifications
 .. This is a reference anchor used in the main index page.
 .. _versioned_notification_samples:
 
-.. versioned_notifications::
 
 
 
